sklearn/metrics/pairwise.py

```diff
@@ -1429,6 +1602,28 @@ def _check_chunk_size(reduced, chunk_size):
 def _precompute_metric_params(X, Y, metric=None, **kwds):
     """Precompute data-derived metric parameters if not provided
     """
+    if metric == 'gower':
+        categorical_features = kwds.get('categorical_features', None)
+
+        _, X_num = _split_categorical_numerical(X, categorical_features)
+        _, Y_num = _split_categorical_numerical(Y, categorical_features)
+
+        scale = kwds.get('scale', True)
+        if not scale:
+            return {'min_values': None, 'scale_factor': None, 'scale': False}
+
+        scale_factor = kwds.get('scale_factor', None)
+        min_values = kwds.get('min_values', None)
+        if min_values is None:
+            data = X_num if Y is X or Y is None else np.vstack((X_num, Y_num))
+            trs = MinMaxScaler().fit(data)
+            min_values = trs.min_
+            scale_factor = trs.scale_
+
+        return {'min_values': min_values,
+                'scale_factor': scale_factor,
+                'scale': True}
```

Comment:
Ideally this should also pass `categorical_features` so it does not need to be recomputed in case of a callable (in which there's the potential for inconsistency)